sqld/src/http/mod.rs

```diff
@@ -159,10 +162,19 @@ async fn show_console() -> anyhow::Result<Response<Body>> {
 }
 
 async fn handle_request(
+    authorizer: Arc<Box<dyn Authorizer + Send + Sync>>,
```

Comment:
`Box` is redundant: `Arc<dyn Authorizer + Send + Sync>` is sufficient

Reply:
fixed